### DISCUSSION TITLE
Auto add bookmark to list

### DISCUSSION THREAD

**Discussion by eliobou** [karakeep-app/karakeep#297] [Ideas]
Hello,

I was wondering if it's possible to have something that auto add the bookmark to a list based on the tag.

Maybe we could set a list of tag attached to this list or even based on a few bookmarks the software could "learn" what should be in this list ?

**Comment by kamtschatka** [comment]
Automatic adding to list is not planned, but we are planning smart lists where you can define a search query. This will then allow you to define a list that selects all bookmarks with a certain tag, which is exactly the outcome you want.
